### DISCUSSION TITLE
How to avoid recurse behavior on directories supplied in `format-resources:`?

### DISCUSSION THREAD

**Discussion by ramiromagno** [quarto-dev/quarto-cli#9874] [Q&A] [answered]
### Description

How to avoid recurse behavior on directories supplied in `format-resources:`?

```
      format-resources:
        - wiley-ndj-v5
        - _environment
```

`wiley-ndj-v5/` is a directory with many files and subdirectories. I would like Quarto not to put everything at the root of the project but just the directory `wiley-ndj-v5/`. Is there a way?

My plan is then to use `_environment` as:

```
TEXINPUTS="wiley-ndj-v5"
```

Related to: https://github.com/quarto-dev/quarto-cli/discussions/6526.

**Comment by mcanouil** [comment]
Do you have a concrete example? A small Git repository maybe?

**Reply by ramiromagno** [reply]
The only example I have is this extension: https://github.com/ramiromagno/wiley-njd/. But I can try to make a smaller extension. See: https://github.com/ramiromagno/wiley-njd/issues/13.

**Comment by cderv** [comment]
It seems like an issue following a change we made... 

File globbing now works in format resources following this change 7d95eb886f161a4cee5572b49b191ce1b084f3e7 to solve #6241

https://github.com/quarto-dev/quarto-cli/blob/e09ae925dd92950394ca7eb2376ca0e3fba34bc5/src/extension/extension.ts#L724-L736 

And this is a 1.4 regression compared to 1.3, because now `format-resources` does not accept directory anymore. This has a big impact on some template. 

Using v1.4.516 will copy the directory and v1.4.517 and later will copy the file inside the directory. 

So I will open an issue about this, because we need to either revert or adapt, or document the changes.

**Reply by cderv** [reply]
Note that #6241 was related to another one of your discussion item @ramiromagno 😉 
* https://github.com/quarto-dev/quarto-cli/discussions/6229 

So we introduced the support following your comment, but it broke this. 😅

**Reply by cderv** [reply, marked as answer]
Tracked at https://github.com/quarto-dev/quarto-cli/issues/9918 now
